### PR TITLE
draco: add cmake/pkgconfig name + libm + no cross build in test_package

### PR DESCRIPTION
**Pull request by SpaceIm**
Specify library name and version:  **draco/all**

- [x] I've read the [guidelines](https://github.com/conan-io/conan-center-index/wiki#how-to-submit-a-pull-request) for contributing.
- [x] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [x] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [x] I've tried at least one configuration locally with the
      [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

File: recipes/draco/all/conanfile.py [hunk 1]
```diff
@@ -49,8 +49,8 @@ def config_options(self):
             del self.options.fPIC
 
     def configure(self):
-        if self.options.shared:
-            del self.options.fPIC
```
Comment (prince-chrismc) [inline recipes/draco/all/conanfile.py:53]: For my knowledge, why was this removed?

Comment (uilianries) [reply recipes/draco/all/conanfile.py:53]: disabling fPIC for shared library is an error, that's why it's removed. The only motivation to keep it for both, is because fPIC is a default option just like shared, and removing it could break users which force fPIC=False by default (we saw it in the past running Conan Package Tools).

Comment (madebr) [reply recipes/draco/all/conanfile.py:53]: Removing that `fPIC` option does not disable fpic. It's just keeps the default.
https://github.com/conan-io/conan/blob/f361690beae3608d93a8d79dec7abb24f75ddd85/conans/client/build/cmake_flags.py#L337-L343
When settings `shared=True` and `fPIC=False`, a build will fail if a shared library includes a static library (that has not overridden the pic flag).

File: recipes/draco/all/conanfile.py [hunk 2]
```diff
@@ -126,4 +125,9 @@ def package(self):
         self.copy(pattern="*.dll", dst="bin", src=build_bin_dir, keep_path=False)
 
     def package_info(self):
+        self.cpp_info.names["cmake_find_package"] = "Draco"
+        self.cpp_info.names["cmake_find_package_multi"] = "Draco"
+        self.cpp_info.names["pkg_config"] = "Draco"
```
Comment (SpaceIm) [inline recipes/draco/all/conanfile.py:130]: Mmhh I think that pkg_config name I gave is wrong. Draco generates `draco.pc` file with `Name` attribute `Draco`, but what is important is the file name, not `Name` attribute in pc file, right?

Comment (madebr) [reply recipes/draco/all/conanfile.py:130]: I think so. What matters is whether `pkg-config` will find the package.

File: recipes/draco/all/conanfile.py (hunk 1 above)
Comment (prince-chrismc) [reply recipes/draco/all/conanfile.py:53]: Ahhh thank you both ! I am pretty sure a recipe I merged has this
